cli/lsp/language_server.rs

```diff
@@ -3614,6 +3614,11 @@ impl Inner {
         }),
         // bit of a hack to force the lsp to cache the @types/node package
         type_check_mode: crate::args::TypeCheckMode::Local,
+        permissions: crate::args::PermissionFlags {
+          // allow remote import permissions in the lsp for now
+          allow_import: Some(vec![]),
```

Comment:
Maybe open an issue about this one and ask @nayeemrmn to add a setting in `vscode_deno` to pass an allow list?